src/components/Icons/Twemoji/Camera.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const SvgTwemoji1F4F7 = props => (
+  <svg viewBox="0 0 47.5 47.5" {...props}>
+    <defs>
+      <clipPath id="Twemoji_1f4f7_svg__a">
+        <path d="M0 38h38V0H0v38z" />
+      </clipPath>
+    </defs>
+    <g
+      clipPath="url(#Twemoji_1f4f7_svg__a)"
+      transform="matrix(1.25 0 0 -1.25 0 47.5)"
+    >
+      <path
+        d="M1 27s0 4 4 4h28s4 0 4-4V9s0-4-4-4H5S1 5 1 9v18z"
+        fill="#292f33"
+      />
+      <path
+        d="M12 18c0-5.523 4.477-10 10-10s10 4.477 10 10-4.477 10-10 10-10-4.477-10-10"
+        fill="#ccd6dd"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)